core/src/main/java/org/frankframework/http/rest/ApiListenerServlet.java

```diff
@@ -514,7 +519,8 @@ private void handleRequest(HttpServletRequest request, HttpServletResponse respo
 				PipeLineSession.updateListenerParameters(pipelineSession, messageId, correlationId);
 
 				/*
-				 * Do the actual request processing by the ApiListener
+				 * Do the actual request processing by the ApiListener.
+				 * Body may not be null!
```

Comment:
Misschien beter dan om de if & assignment dan hier neer te zetten ipv daarboven?

Reply:
Hmm, misschien wel handig, maar aan de andere kant moeten we het denk ik op adapter/pipeline niveau op gaan lossen. Voor nu hebben we hier iig een test voor :)